test/unit/custom-vocabularies-client.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { ApiRequestHandler } from '../../src/api-request-handler';
+import { RevAiCustomVocabulariesClient } from '../../src/custom-vocabularies-client';
+// import { RevAiApiTranscript } from '../../src/models/RevAiApiTranscript';
+
+import { objectToStream } from './testhelpers';
+
+jest.mock('../../src/api-request-handler');
+
+let sut: RevAiCustomVocabulariesClient;
+
+describe('api-client', () => {
+    const customVocabularies = [
+        {phrases: ['my', 'test', 'custom', 'vocabularies']}
+    ];
+    const callbackUrl = 'dummy.dummy';
+    const metadata = 'my metadata';
+    const customVocabularyOptions = {
+        custom_vocabularies: customVocabularies,
+        callback_url: callbackUrl,
+        metadata: metadata
+    };
+
+    const customVocabularyId = 'Umx5c6F7pH7r';
```

Comment:
use a fake id, nothing real

Reply:
afaik this is fake, I just copied it over from the api-client tests.